go-chaos/internal/network.go

```diff
@@ -15,150 +15,41 @@
 package internal
 
 import (
-	"bytes"
-	"context"
-	"errors"
-	"fmt"
 	"strings"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 )
 
-func (c K8Client) ApplyNetworkPatch() error {
-
-	statefulSet, err := c.GetZeebeStatefulSet()
-	if err != nil {
-		return err
-	}
-
-	// We need to run the container with root to allow install tooling and give it network admin capabilities
-	patch := []byte(`{
-		"spec":{
-			"template":{
-				"spec":{
-					"containers":[
-						{
-							"name": "zeebe",
-							"securityContext":{
-								"runAsUser": 0,
-								"capabilities":{
-									"add":["NET_ADMIN"]
-								}
-							}
-						}]
-				}
-			}
-		}
-	}`)
-
-	_, err = c.Clientset.AppsV1().StatefulSets(c.GetCurrentNamespace()).Patch(context.TODO(), statefulSet.Name, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
-	return err
-}
-
-func (c K8Client) ApplyNetworkPatchOnGateway() error {
-
-	deployment, err := c.getGatewayDeployment()
-	if err != nil {
-		return err
-	}
-
-	// We need to run the container with root to allow install tooling and give it network admin capabilities
-	patch := []byte(`{
-		"spec":{
-			"template":{
-				"spec":{
-					"containers":[
-						{
-							"name": "zeebe-gateway",
-							"securityContext":{
-								"runAsUser": 0,
-								"capabilities":{
-									"add":["NET_ADMIN"]
-								}
-							}
-						}]
-				}
-			}
-		}
-	}`)
-
-	_, err = c.Clientset.AppsV1().Deployments(c.GetCurrentNamespace()).Patch(context.TODO(), deployment.Name, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
-	return err
-}
-
 func MakeIpUnreachableForPod(k8Client K8Client, podIp string, podName string) error {
-	// We try to reduce the system output in order to not break the execution. There is a limit for the sout for exec,
-	// for more details see remotecommand.StreamOptions
-
-	// the -qq flag makes the tool less noisy, remove it to get more output
```

Comment:
❓ Do you have seen the comments? I think this still applies?

Reply:
We could still use `-qq` but it doesn't hurt to have more output. We are not capturing stdout anymore, it's only visible through the container logs (`kubectl logs ...`)